app/server.go

```diff
@@ -1070,6 +1077,11 @@ func (s *Server) Shutdown() {
 
 	mlog.Info("Server stopped")
 
+	mlog.Debug("Stopping focalboard...")
+	if err := s.fbServer.Shutdown(); err != nil {
+		mlog.Critical("Error starting focalboard", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.